packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -351,6 +358,53 @@ describe.each( [
 
 				expect( expectedFocusTarget ).toHaveFocus();
 			} );
+
+			it( 'should ignore disabled radio options', async () => {
+				const mockOnChange = jest.fn();
+
+				render(
+					<Component
+						value="pizza"
+						onChange={ mockOnChange }
+						label="Test Toggle Group Control"
+					>
+						{ optionsWithDisabledOption }
+					</Component>
+				);
+
+				await sleep();
+				await press.Tab();
+
+				expect(
+					screen.getByRole( 'radio', { name: 'Pizza' } )
+				).toBeChecked();
+				expect(
+					screen.getByRole( 'radio', { name: 'Rice' } )
+				).toBeDisabled();
```

Comment:
This assertion seems redundant to me since it's testing an unrelated feature (whether passing `disabled` to a toggle group control option disables that option).

Reply:
I've updated tests to reflext the new "accessible when disabled" behavior, assertions have changed a bit. I will mark this as resolved, but feel free to add new comments as necessary